### PR TITLE
FRONT001: add base react setup - add AddNewUser component - add Users…

### PR DESCRIPTION
**Pull request by dBanasiak**
…List component

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import './App.scss';
+import UsersList from './views/UsersList';
+
+function App() {
+	return (
```
Comment (KonradLinkowski) [inline frontend/src/App.js:5]: Indentation with tabs again

File: frontend/src/App.test.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  	render(<App />);
```
Comment (KonradLinkowski) [inline frontend/src/App.test.js:5]: Mixed indentation - 2 spaces and 1 tab to be precise

File: frontend/src/views/AddNewUser.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import socketIOClient from 'socket.io-client';
+
+const AddNewUser = () => {
+    const [user, setUser] = useState('');
+	const [isVisible, setIsVisible] = useState(true);
```
Comment (KonradLinkowski) [inline frontend/src/views/AddNewUser.js:6]: Add a linter please xd

**Review by KonradLinkowski — COMMENTED** [review]
I would add eslint or prettier to fix these mixed indentations

File: frontend/src/views/AddNewUser.js (hunk 3 above)
Comment (dBanasiak) [reply frontend/src/views/AddNewUser.js:6]: I will add in the next  PR

**Pull request merged**